css-three-column-layout/styles.css

```diff
@@ -0,0 +1,69 @@
+body {
+  background-color: #1996da;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+  justify-content: center;
+  align-items: center;
+  max-height: 250px;
```

Comment:
The `justify-content: center` and `align-items: center` don't do anything here. These cannot only be applied to a `flex container` element. This `column-one-third` is a `flex item`.

Also, your utility classes should do one job. The `column-one-third` class should only give a `flex-basis: calc(100% / 3)`